src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number,
+  userId: number,
+  title: string,
+  completed: boolean,
+  user: User | null,
```

Comment:
Consider using optional chaining instead of `user: User | null` for better readability and maintainability. 🤓